python/ray/worker.py

```diff
@@ -236,6 +236,22 @@ def __init__(self):
         # CUDA_VISIBLE_DEVICES environment variable.
         self.original_gpu_ids = ray.utils.get_cuda_visible_devices()
         self.profiler = profiling.Profiler(self)
+        # A dictionary that maps from driver id to SerializationContext
+        # TODO: clean up the SerializationContext once the job finished.
+        self.serialization_context_map = dict()
+        # Identity of the driver that this worker is processing.
+        self.task_driver_id = None
```

Comment:
This `None` value is never used, right?

Reply:
I put it here for better readability, people will easily know we have a task_driver_id which could be accessed in this way: work.task_driver_id.